internal/cloud/azure/metadata_test.go

```diff
@@ -650,6 +653,10 @@ func newScaleSetsStub() *stubScaleSetsAPI {
 		pager: &stubVirtualMachineScaleSetsClientListPager{
 			list: []armcomputev2.VirtualMachineScaleSet{{
 				Name: to.Ptr("scale-set-name"),
+				Tags: map[string]*string{
+					"uid":  to.Ptr("uid"),
```

Comment:
It's `constellation-uid`. Also change in `azure/metadata.go:303`. Cannot currently init because of nil pointer exception. Probably, also add a check for that in `azure/metadata.go:300`.